case_studies/weather_dependent_app_example/appserver/static/javascript/views/setup_page.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+import * as Splunk from './splunk_helpers.js'
+import * as Config from './setup_configuration.js'
+import * as StoragePasswords from './storage_passwords.js'
+
+const CUSTOM_CONF = 'weather'
+const CUSTOM_CONF_STANZA = 'api'
+
+const SECRET_REALM = 'weather_dependent_app_realm'
+const SECRET_NAME = 'admin'
+
+export async function perform(splunk_js_sdk, setup_options) {
+    var app_name = "weather_dependent_app_example";
+
+    var application_name_space = {
+        owner: "nobody",
+        app: app_name,
+        sharing: "app",
+    };
+
+    try {
+        // Create the Splunk JS SDK Service object
+        const splunk_js_sdk_service = Config.create_splunk_js_sdk_service(
+            splunk_js_sdk,
+            application_name_space,
+        );
+
+        let configured = false
+        const dependent_app_name = "weather_app_example"
+
+        var apps = await Config.list_apps(splunk_js_sdk_service);
+        var current_app = apps.item(dependent_app_name);
+       
+        if (current_app.properties().configured == true) {
+            configured = true
+        } else {
+            alert('Error:' + 'Dependent weather_app_example is not installed!');
```

Comment:
"The weather-app-example dependency is not installed. Install and configure the weather-app-example, then try again."

Reply:
Updated.